### ISSUE TITLE
Deprecate WMF image

### ISSUE THREAD

**Issue by FranklinYu** [chocolatey-community/chocolatey-packages-template#27]
1. It's not in [the list of images in documentation](https://www.appveyor.com/docs/build-environment/).
2. It is deprecated according to [this post](https://help.appveyor.com/discussions/questions/2433-list-of-os-images):

    > `WMF 5` - it's VS 2015 image + WMF 5.1 Preview on top of it. Will be deprecated to the public soon and provided by request.

**Comment by franklinyu** [comment]
Even the default image seems to work: <https://ci.appveyor.com/project/chocolatey/chocolatey-coreteampackages/build/3849>

**Comment by AdmiringWorm** [comment]
yup, the default image indeed do work and without any increase in the time it takes to run the build (unlike earlier versions of the image).
